priv/templates/nimble_template/.github/workflows/test.yml.eex

```diff
@@ -88,6 +88,15 @@ jobs:
       - name: Run codebase check
         run: mix codebase
 
+      - name: Ensure that localization files (POs, POTs) are up-to-date.
```

Comment:
@byhbt do we need this step as we have the `gettext.check` in the `mix codebase` above 🤔

Reply:
right, it's duplicated, i removed it https://github.com/nimblehq/elixir-templates/pull/138/commits/139ab3f040af54805e54be98656be3b0e07dc371